src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -21,11 +25,21 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     }
 
     @Override
-    public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException, ServletException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
-        driverService.create(driver);
-        resp.sendRedirect(req.getContextPath() + "/drivers/add");
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        String repeatPassword = req.getParameter("repeat_password");
+        try {
+            validationService.validatePassword(password, repeatPassword);
```

Comment:
```suggestion
        try {
```
remove this for now, and remove validation services